### PR TITLE
Replace log package with logrus

### PR DESCRIPTION
**Pull request by saschagrunert**
We now introduce a new `--log-level` flag to set the appropriate level, whereas the usage of the internal log package has been replaces with logrus.

### CODE REVIEW

**Comment by hasheddan** [comment]
@saschagrunert I have the updated `push.go` with `logrus` updated coming shortly in the follow-up PR if you want to exclude to avoid rebasing :) Totally fine either way!

**Comment by saschagrunert** [comment]
> @saschagrunert I have the updated `push.go` with `logrus` updated coming shortly in the follow-up PR if you want to exclude to avoid rebasing :) Totally fine either way!

I skipped `push.go` in this PR to not get in conflict with your changes :)

File: cmd/krel/cmd/ff.go [hunk 1]
```diff
@@ -18,13 +18,13 @@ package cmd
...
-
-	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
 
 	kgit "k8s.io/release/pkg/git"
 	"k8s.io/release/pkg/util"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
```
[the remark this answers is not in the document]
Comment (cpanato) [reply cmd/krel/cmd/ff.go]: good to know, there is any place that we can read the convention?

[the remark this answers is not in the document]
Comment (saschagrunert) [reply cmd/krel/cmd/ff.go]: This one? https://github.com/golang/go/wiki/CodeReviewComments#imports

Some time ago the goimports linter complained about this (via golangci-lint). But this does not seem to apply any more.

[the remark this answers is not in the document]
Comment (cpanato) [reply cmd/krel/cmd/ff.go]: ahh :)

[the remark this answers is not in the document]
Comment (justaugustus) [reply cmd/krel/cmd/ff.go]: Hmmm, I can't seem to find it here: https://github.com/kubernetes/community/blob/master/contributors/guide/coding-conventions.md

...but it's something Jordan told me once.

**Comment by justaugustus** [comment]
/lgtm
/approve

The import fix is cosmetic and can be handled as a follow-up.

**Comment by cpanato** [comment]
@justaugustus doing the follow up on this

**Pull request merged**